homeassistant/components/vicare/binary_sensor.py

```diff
@@ -102,6 +103,16 @@ class ViCareBinarySensorEntityDescription(
     ),
 )
 
+STATUS_SENSOR: ViCareBinarySensorEntityDescription = (
+    ViCareBinarySensorEntityDescription(
+        key="status",
+        name="Status",
```

Comment:
Since ViCare uses `has_entity_name = True` I would suggest to remove the name, it will fallback to the device class name

Reply:
This would mean I need to rework the base class.